torchvision/transforms.py

```diff
@@ -222,11 +309,8 @@ def __call__(self, img):
         Returns:
             PIL.Image: Cropped image.
         """
-        w, h = img.size
-        th, tw = self.size
-        x1 = int(round((w - tw) / 2.))
-        y1 = int(round((h - th) / 2.))
-        return img.crop((x1, y1, x1 + tw, y1 + th))
+        x1, y1, tw, th = self.get_params(img)
+        return crop(img, x1, y1, tw, th)
```

Comment:
I think if we want a complete functional interface we should have a `center_crop` function which has this logic and then calls the generic `crop` function

Reply:
Then what about `random_crop`? On one hand, having one function for `random_crop` will defeat the whole purpose. One the other hand, it'll be inconsistent because `center_crop` exists.

Reply:
I think it depends on whether we factor out the random parameters to a function like `get_random_crop_params`, if so then `random_crop` doesn't make much sense as it's just this + `crop`. cc @fmassa what are your thoughts?

Reply:
I think for the moment let's keep the functional interface very simple, and only let `crop` have a functional form, and not `center_crop` etc.
It's easier to add new functions than to remove them later.